packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.Props.ts

```diff
@@ -15,6 +15,10 @@ export interface ISuggestionsProps<T> extends React.Props<any> {
    */
   suggestions: ISuggestionModel<T>[];
   /**
+  * How the "no result found" should look in the suggestion list.
+  */
+  onRenderNoResultFound?: () => JSX.Element;
```

Comment:
ping. If you'd like a bit more time to fix this, you can always close and then resubmit later.